src/converse-muc-views.js

```diff
@@ -82,6 +82,8 @@
         'unmoderated': 'moderated'
     };
 
+    Strophe.addNamespace('OUTOFBAND', 'jabber:x:oob');
```

Comment:
This namespace is declared twice, we might as well rather declare it once in `converse-core.js` or perhaps in `converse-chatboxes.js`.